pages/releases/4.3.html

```diff
@@ -0,0 +1,2360 @@
+							</video>
+						</div>
+					</div>
+				</div> <!-- #2d-pixel-art-stability -->
+			</div>
+		</div>
+	</div> <!-- #2d -->
+
+	<div id="rendering" class="section section-rendering">
+		<div class="section-title">
+			<div class="container">
+				<h3>
+					<a href="#rendering">Rendering</a>
+					<div class="section-robot godot-icon-godot" aria-disabled="true"></div>
+				</h3>
+			</div>
+		</div>
+		<div class="container">
+			<div class="release-cards">
+				<div id="rendering-premultiplied-alpha-blending" class="release-card media-top inverted span-2">
+					<div class="release-card-container">
+						<div class="release-card-content">
+							<div class="release-card-content-container">
+								<h4 class="c-title">
+									<a href="#rendering-premultiplied-alpha-blending">
+										Premultiplied alpha blending in 3D shaders
+									</a>
+								</h4>
+								<div class="c-content">
+									<p>
+										Premultiplied alpha is a new blend mode for 3D materials. This enables you to
+										create better looking flames and fireworks, as the same particle now can have
+										the properties of additive blending (for the flame) and mix blending (for the
+										smoke) blending.
+									</p>
+									<p>
+										See in the video how the left barrel flames (which uses additive blending)
+										are missing the smoke that is visible in the other barrel flames which use
+										premultiplied alpha blending.
+									</p>
+								</div>
+								<div class="c-link"
+									data-contributors="QbieShay,jitspoe,clayjohn"
+									data-read-more="https://github.com/godotengine/godot/pull/85609">
+								</div>
+							</div>
+						</div>
+						<div class="release-card-media">
+							<video class="release-card-video lazy" playsinline disablepictureinpicture autoplay loop muted
+								poster="/storage/releases/4.3/images/premultiplied-alpha.webp">
+								<source data-src="/storage/releases/4.3/video/premultiplied-alpha.webm" type="video/webm">
+							</video>
+						</div>
+					</div>
+				</div> <!-- #rendering-premultiplied-alpha-blending -->
+
+				<div id="rendering-compositor-effects-api" class="release-card">
+					<div class="release-card-container">
+						<div class="release-card-content">
+							<div class="release-card-content-container">
+								<h4 class="c-title">
+									<a href="#rendering-compositor-effects-api">Compositor effects API</a>
+								</h4>
+								<div class="c-content">
+									<p>
+										Compositor effects let users hook into the built in Godot rendering pipeline
+										to add custom rendering code. This is an advanced feature that does require
+										intimate knowledge of how rendering works. Yet, it enables those specialists
+										to create a variety of effects more efficiently.
+									</p>
+									<p>
+										We're already seeing some great examples from early adaptors who utilize this
```

Comment:
```suggestion
										We're already seeing some great examples from early adopters who utilize this
```
Just a small typo

Reply:
Credit where credit is due, your texts needed the least amount of change, they were already very comprehensible!